scripts/download_osm_data.py

```diff
@@ -96,7 +129,12 @@ def convert_iso_to_geofk(
     out_path = Path.joinpath(store_path_resources, "out")
     out_formats = ["csv", "geojson"]
     new_files = os.listdir(out_path)
-
+    microgrid_coordinates = snakemake.config["microgrids_list"]["microgrid_1"]
+    features = "building"
+    overpass_url = "https://overpass-api.de/api/interpreter"
+    retrieve_osm_data(
```

Comment:
Currently, the retrieve_osm_data function is executed after the standard approach.
It would be great to have an option and execute either the standard apporach or the novel one using overpass-api